crawler/emitter.py

```diff
@@ -241,6 +240,61 @@ def _publish_to_stdout(self):
                     print line.strip()
                     sys.stdout.flush()
 
+    def _publish_to_logstash(self, url, max_emit_retries=5):
```

Comment:
this is just regular http right? why make it logstash specific. BTW you need add unit tests in tests/unit/test_emitter.py for this new target.